src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -57,6 +58,9 @@
   {:render
    (fn [{:keys [props]}]
      [:div {:style {:textAlign "right"}}
+      [TopNavBarLink {:name "Import Data"
```

Comment:
So if you're adding something to the top nav, you have to do so in three places: here, line 77, and line 96. Can we make a list of "things to put in the top nav" and reference it from these three places instead? 'cos I missed one first time round.